test/01_cond_field/process.py

```diff
@@ -1,194 +1,111 @@
 import os
 import sys
+import shutil
 import ruamel.yaml as yaml
+import numpy as np
 
-src_path = os.path.dirname(os.path.abspath(__file__))
-sys.path.append(os.path.join(src_path, '..', '..', 'src'))
-src_path = os.path.dirname(os.path.abspath(__file__))
-sys.path.append(os.path.join(src_path, '..', 'src/mlmc'))
-
-import flow_mc as flow_mc
-from moments import Legendre
-from sampler import Sampler
-from sample_storage import Memory
-from sample_storage_hdf import SampleStorageHDF
-from sampling_pool_pbs import SamplingPoolPBS
-from quantity_estimate import QuantityEstimate
-import new_estimator
-
-
-
-sys.path.append(os.path.join(src_path, '..'))
-import base_process
-
-
-class FlowProcSim(flow_mc.FlowSim):
-    """
-    Child from FlowSimulation that defines extract method
-    """
-
-    @staticmethod
-    def _extract_result(sample_dir):
-        """
-        Extract the observed value from the Flow123d output.
-        :param sample: str, path to sample directory
-        :return: None, inf or water balance result (float) and overall sample time
-        """
-        if os.path.exists(os.path.join(sample_dir, "FINISHED")):
-            # try:
-            # extract the flux
-            balance_file = os.path.join(sample_dir, "water_balance.yaml")
-
-            with open(balance_file, "r") as f:
-                balance = yaml.load(f)
-
-            flux_regions = ['.bc_outflow']
-            total_flux = 0.0
-            found = False
-            for flux_item in balance['data']:
-                if flux_item['time'] > 0:
-                    break
-
-                if flux_item['region'] in flux_regions:
-                    flux = float(flux_item['data'][0])
-                    flux_in = float(flux_item['data'][1])
-                    if flux_in > 1e-10:
-                        raise Exception("Possitive inflow at outlet region.")
-                    total_flux += flux  # flux field
-                    found = True
-
-            # Get flow123d computing time
-            run_time = self.get_run_time(sample_dir)
-
-            if not found:
-                raise Exception
-            return -total_flux, run_time
-        else:
-            return None, 0
+from mlmc.random import correlated_field as cf
+from mlmc.tool import flow_mc
+from mlmc.moments import Legendre
+from mlmc.sampler import Sampler
+from mlmc.sample_storage_hdf import SampleStorageHDF
+from mlmc.sampling_pool import OneProcessPool, ProcessPool, ThreadPool
+from mlmc.sampling_pool_pbs import SamplingPoolPBS
+from mlmc.tool import base_process
+from mlmc.tool.flow_mc import FlowSim
+from mlmc.moments import Legendre
+from mlmc.quantity_estimate import QuantityEstimate
 
 
 class CondField(base_process.Process):
 
```

Comment:
`01_cond_field/process.py` should work as a basic tutorial of using MLMC with Flow123d, to this end I propose
to remove support for testing the different number of levels in single run. That way we can make usage of ProcessBase more straightforward.